packages/@apphosting/adapter-nextjs/src/interfaces.ts

```diff
@@ -74,9 +74,29 @@ export interface RoutesManifest {
 
 // The output bundle options are specified here
 export interface OutputBundleOptions {
+  /**
+   * Path to the generated bundle.yaml file
+   */
   bundleYamlPath: string;
-  outputDirectory: string;
+  /**
+   *  Path to where the base output directory (.apphosting) is generated
+   */
+  outputDirectoryBasePath: string;
+  /**
+   * Path to where the App's code sits in the output directory.
+   * This will only be different from outputDirectoryPath in Monorepo scenerios.
```

Comment:
This may be a dumb question but are these monorepo scenarios obvious? Should we have an example?

Reply:
There's a comment that explains this more where this is set https://github.com/FirebaseExtended/firebase-framework-tools/blob/main/packages/%40apphosting/adapter-nextjs/src/utils.ts#L56. I also added an example to make this more clear here